### ISSUE TITLE
Request table with crafting monitor shows frozen status even craft request successfully done.

### ISSUE THREAD

**Issue by ZZaphodd** [RS485/LogisticsPipes#970]
LP version : 0.9.3.114
Crafting done but not gone,
![image](https://cloud.githubusercontent.com/assets/8099421/14579302/cd3a05ba-03de-11e6-984f-2f60bc8809dd.png)

"More..." buttons also frozen
![image](https://cloud.githubusercontent.com/assets/8099421/14579289/63cd6ea0-03de-11e6-8b14-5fd8e25b6e09.png)

(btw, It's first time to see auto crafting status tree, It's stunning and awesome)

**Comment by ZZaphodd** [comment]
And still showing frozen status even player issues additional craft request or close ui of the request table open again. Additional craft request itself seems works properly.

**Comment by SonOfTheStars** [comment]
For now, did you try removing the Montioring Upgrade and putting it back in?

**Comment by ZZaphodd** [comment]
yes I tried removing and putting the upgrade back, 'frozen status' are gone but didn't shows current crafting task.

I tried pick the request table up and connect to network again, still not showing the current crafting task.
Restarting the game won't fixes it

Resetting whole netework fixes this problem.(break and place again the power junction)

**Comment by davboecki** [comment]
Did you try to simply reconnect to the server / reload the world if it was SSP?

**Comment by davboecki** [comment]
@ZZaphodd Does your server log show something like:
https://gist.github.com/davboecki/527a7d14a87f6e63104217bb186a55be ?

**Comment by ZZaphodd** [comment]
It's pretty hard to reproduce. I'm going to upload server log when I see this behavior again.

btw, where can I find 'server log'? I'm playing single player game, and I see several types of log seems like client and 'forge' or something, but I cannot find seems like server log. 
(As far as I know, single player game is actuallly
mutli player game opened over lan)

**Comment by ZZaphodd** [comment]
Hm.. it seems be logger verbose level set as Minimal on ftb launcher.
I changed it as debug and I'm going to post log as soon as possible

**Comment by Darklight-03** [comment]
I am having the same issue on my ftb infinity evolved server, also version 0.9.3.114. I get this in my console, which seems similar to what @davboecki posted earlier. I am unsure as to when this issue started occurring however, so I don't know if that came up at the same time as the request monitor stopped working. Reconnecting to the server does not fix the issue, but restarting the entire server probably does, I haven't tried that yet.

```
[00:45:40] [LogisticsPipes Packet Compressor Server/INFO] [STDERR/LogisticsPipes]: [java.lang.Throwable$WrappedPrintStream:println:748]: java.lang.NullPointerException
[00:45:40] [LogisticsPipes Packet Compressor Server/INFO] [STDERR/LogisticsPipes]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at logisticspipes.routing.order.LogisticsOrder.getTargetType(LogisticsOrder.java:97)
[00:45:40] [LogisticsPipes Packet Compressor Server/INFO] [STDERR/LogisticsPipes]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at logisticspipes.network.LPDataOutputStream.writeOrderInfo(LPDataOutputStream.java:188)
[00:45:40] [LogisticsPipes Packet Compressor Server/INFO] [STDERR/LogisticsPipes]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at logisticspipes.network.LPDataOutputStream$3.writeObject(LPDataOutputStream.java:200)
[00:45:40] [LogisticsPipes Packet Compressor Server/INFO] [STDERR/LogisticsPipes]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at logisticspipes.network.LPDataOutputStream$3.writeObject(LPDataOutputStream.java:196)
[00:45:40] [LogisticsPipes Packet Compressor Server/INFO] [STDERR/LogisticsPipes]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at logisticspipes.network.LPDataOutputStream.writeList(LPDataOutputStream.java:165)
[00:45:40] [LogisticsPipes Packet Compressor Server/INFO] [STDERR/LogisticsPipes]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at logisticspipes.network.LPDataOutputStream.writeLinkedLogisticsOrderList(LPDataOutputStream.java:196)
[00:45:40] [LogisticsPipes Packet Compressor Server/INFO] [STDERR/LogisticsPipes]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at logisticspipes.network.LPDataOutputStream$4.writeObject(LPDataOutputStream.java:207)
[00:45:40] [LogisticsPipes Packet Compressor Server/INFO] [STDERR/LogisticsPipes]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at logisticspipes.network.LPDataOutputStream$4.writeObject(LPDataOutputStream.java:203)
[00:45:40] [LogisticsPipes Packet Compressor Server/INFO] [STDERR/LogisticsPipes]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at logisticspipes.network.LPDataOutputStream.writeList(LPDataOutputStream.java:165)
[00:45:40] [LogisticsPipes Packet Compressor Server/INFO] [STDERR/LogisticsPipes]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at logisticspipes.network.LPDataOutputStream.writeLinkedLogisticsOrderList(LPDataOutputStream.java:203)
[00:45:40] [LogisticsPipes Packet Compressor Server/INFO] [STDERR/LogisticsPipes]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at logisticspipes.network.packets.orderer.OrdererWatchPacket.writeData(OrdererWatchPacket.java:39)
[00:45:40] [LogisticsPipes Packet Compressor Server/INFO] [STDERR/LogisticsPipes]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at logisticspipes.ticks.ServerPacketBufferHandlerThread$ServerCompressorThread.run(ServerPacketBufferHandlerThread.java:66)
```

**Comment by bziemons** [comment]
Should be fixed in 0.9.3.130
